maintainers/maintainer-list.nix

```diff
@@ -5973,7 +5973,7 @@
   fortuneteller2k = {
     email = "lythe1107@gmail.com";
     matrix = "@fortuneteller2k:matrix.org";
-    github = "fortuneteller2k";
+    github = "moni-dz";
     githubId = 20619776;
     name = "fortuneteller2k";
   };
```

Comment:
https://github.com/NixOS/nixpkgs/pull/268077

Reply:
name should be `"moni"` as well